src/App.vue

```diff
@@ -104,7 +104,13 @@
 
     <SbcHeader />
 
-    <PaySystemAlert />
+    <!-- Alert banner -->
+    <v-alert
+      tile dense
+      type="warning"
+      v-if="bannerText">
+      <div v-html="bannerText" class="mb-0 text-center colour-dk-text"></div>
```

Comment:
Did you test this? I'm wondering if `colour-dk-test` is declared in the Vuetify themes; you may need to add it.

Reply:
I haven't test yet. I added this into base.scss .